elasticsearch5/src/main/java/org/vertexium/elasticsearch5/ElasticsearchSearchQueryBase.java

```diff
@@ -160,6 +160,9 @@ private SearchRequestBuilder buildQuery(EnumSet<ElasticsearchDocumentType> eleme
                 .storedFields(
                         Elasticsearch5SearchIndex.ELEMENT_ID_FIELD_NAME,
```

Comment:
Might want to be smarter about this list of fields to return. We could look at the fetch hints and choose only the fields we need. Bringing back in, out, and label when we aren't going to use them might be slow.